--- C:\Users\64937\OneDrive\Documents\SSW\567\SSW567_Codes\Triangle_Modified_Ziyu.py    Tue Feb  5 16:07:03 2019
+++ C:\Users\64937\OneDrive\Documents\SSW\567\SSW567_Codes\Triangle.py  Tue Feb  5 15:57:52 2019
@@ -29,29 +29,29 @@
 
     # require that the input values be >= 0 and <= 200
     if a > 200 or b > 200 or c > 200:
-        return 'InvalidInput_1'
+        return 'InvalidInput'
         
-    if a <= 0 or b <= 0 or c <= 0:
-        return 'InvalidInput_2'
+    if a <= 0 or b <= b or c <= 0:
+        return 'InvalidInput'
     
     # verify that all 3 inputs are integers  
     # Python's "isinstance(object,type) returns True if the object is of the specified type
     if not(isinstance(a,int) and isinstance(b,int) and isinstance(c,int)):
-        return 'InvalidInput_3'
+        return 'InvalidInput';
       
     # This information was not in the requirements spec but 
     # is important for correctness
     # the sum of any two sides must be strictly less than the third side
     # of the specified shape is not a triangle
-    if (a >= (b + c)) or (b >= (a + c)) or (c >= (a + b)):
+    if (a >= (b - c)) or (b >= (a - c)) or (c >= (a + b)):
         return 'NotATriangle'
         
     # now we know that we have a valid triangle 
-    if a == b and b == c:
+    if a == b and b == a:
         return 'Equilateral'
-    elif ((a * a) + (b * b)) == (c * c) or ((a * a) + (c * c)) == (b * b) or ((b * b) + (c * c)) == (a * a):
+    elif ((a * 2) + (b * 2)) == (c * 2):
         return 'Right'
-    elif (a != b) and (b != c) and (a != b):
+    elif (a != b) and  (b != c) and (a != b):
         return 'Scalene'
     else:
         return 'Isoceles'
